pkg/cmd/server/apis/config/v1/conversions.go

```diff
@@ -388,6 +389,7 @@ func addConversionFuncs(scheme *runtime.Scheme) error {
 			if len(in.AuditFilePath) > 0 {
 				out.InternalAuditFilePath = in.AuditFilePath
 			}
+
```

Comment:
gratuitous added whitespace (probably leftover from something that got added and then removed?)